src/ProcessSlave.php

```diff
@@ -425,7 +425,18 @@ protected function handleRequest(ServerRequestInterface $request)
         }
 
         if ($bridge = $this->getBridge()) {
-            $response = $bridge->handle($request);
+            try {
+                $response = $bridge->handle($request);
+            } catch (\Exception $exception) {
```

Comment:
You should add a `\Throwable` block for php7 (it's still compatible with php5)

Reply:
Its kind of a hack but sure, I don't think there is another way as long as PHP5 is supported